.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-13]
```

Comment:
`macos-latest` is `macos-12`, are there many differences between `macos-12` and `macos-13` that constitute having both? I'd rather just have `macos-latest`.

Reply:
The biggest difference is that macos-latest is x86 while macos-13 is ARM. So I tend to use both to make sure we're not using SSE/AVX in someplace. Not that we do that here